packages/instrumentation-console/src/index.ts

```diff
@@ -10,19 +10,23 @@ export default function getConsoleInstrumentation({
 }: ConsoleInstrumentationOptions = {}): Instrumentation {
   const defaultDisabledLevels: LogLevel[] = [LogLevel.DEBUG, LogLevel.LOG];
 
-  return () => {
-    allLogLevels
-      .filter((level) => !(disabledLevels ?? defaultDisabledLevels).includes(level))
-      .forEach((level) => {
-        /* eslint-disable-next-line no-console */
-        console[level] = (...args) => {
-          try {
-            agent.api.pushLog(args, level);
-          } catch (err) {
-          } finally {
-            agent.api.callOriginalConsoleMethod(level, ...args);
-          }
-        };
-      });
+  return {
+    initialize: () => {
+      allLogLevels
+        .filter((level) => !(disabledLevels ?? defaultDisabledLevels).includes(level))
+        .forEach((level) => {
+          /* eslint-disable-next-line no-console */
+          console[level] = (...args) => {
+            try {
+              agent.api.pushLog(args, level);
+            } catch (err) {
+            } finally {
+              agent.api.callOriginalConsoleMethod(level, ...args);
+            }
+          };
+        });
+    },
+    version: '1.0',
```

Comment:
Version set similar to how we do it in meta initialization.

Reply:
I added `prebuild` scripts that dump version from `package.json` to `src/version.ts` and then use that